command/sub8_missions/missions/align_path_marker.py

```diff
@@ -10,45 +11,57 @@
 import numpy as np
 
 SEARCH_DEPTH = .65
+SEARCH_RADIUS_METERS = 1.0
 TIMEOUT_SECONDS = 60
+DO_PATTERN=True
+SPEED = 0.5
 MISSION="Align Path Marker"
+
+forward_vec = np.array([1, 0, 0, 0])
 @util.cancellableInlineCallbacks
 def run(sub):
-    print_info = text_effects.FprintFactory(title=MISSION)
-    print_bad = text_effects.FprintFactory(title=MISSION, msg_color="red")
-    print_good = text_effects.FprintFactory(title=MISSION, msg_color="green")
-    print_info.fprint("STARTING")
+    print_info = text_effects.FprintFactory(title=MISSION).fprint
+    print_bad = text_effects.FprintFactory(title=MISSION, msg_color="red").fprint
+    print_good = text_effects.FprintFactory(title=MISSION, msg_color="green").fprint
+    print_info("STARTING")
 
     # Wait for vision services, enable perception
-    print_info.fprint("ACTIVATING PERCEPTION SERVICE")
+    print_info("ACTIVATING PERCEPTION SERVICE")
     sub.vision_proxies.path_marker.start()
 
     pattern = []
-    #pattern = [sub.move.right(1), sub.move.forward(1), sub.move.left(1), sub.move.backward(1),
-     #          sub.move.right(2), sub.move.forward(2), sub.move.left(2), sub.move.backward(2)]
+    if DO_PATTERN:
+        pattern = [sub.move.right(SEARCH_RADIUS_METERS), sub.move.forward(SEARCH_RADIUS_METERS),
```

Comment:
Just realized these moves will all be defined from the same origin, not relative to each other. Will correct